screens/HomeScreen.js

```diff
@@ -0,0 +1,228 @@
+import { View, Text, StyleSheet, ScrollView, Pressable } from 'react-native';
+import { useNavigation } from '@react-navigation/native';
+import CategoryNavigate from '../components/CategoryNavigate';
+
+function HomeScreen(){
+
+    return (
+        <ScrollView showsVerticalScrollIndicator={false} bounces={false}>
+          <View style = {styles.root}>
+            <View style = {styles.first}>
+                <View style = {styles.heading}>
+                    <Text style = {styles.title}>Bestsellers and popular items</Text>
+                </View>
+                <View style = {styles.options}>
+                <ScrollView 
```

Comment:
style issue：Image is cropped

![image](https://github.com/tusharchahar/aaxis/assets/40992422/61a71949-8ea9-4259-b3d3-1e549fecd120)

![image](https://github.com/tusharchahar/aaxis/assets/40992422/ec8e8784-2c56-439d-bde8-d2c26fb63c18)